tests/src/test/resources/kg/archives/archive-response.json

```diff
@@ -1,37 +1,32 @@
 {
-  "@context" : [
-    "https://bluebrain.github.io/nexus/contexts/archive.json",
-    "https://bluebrain.github.io/nexus/contexts/resource.json"
+  "@context": [
+    "https://bluebrain.github.io/nexus/contexts/archives.json",
+    "https://bluebrain.github.io/nexus/contexts/metadata.json"
   ],
-  "@id" : "https://dev.nexus.test.com/simplified-resource/archive",
-  "@type" : "Archive",
-  "resources" : [
+  "@id": "https://dev.nexus.test.com/simplified-resource/archive",
+  "@type": "Archive",
+  "resources": [
     {
-      "@type" : "Resource",
-      "originalSource" : true,
-      "project" : "{{project2}}",
-      "resourceId" : "https://dev.nexus.test.com/simplified-resource/2"
+      "@type": "Resource",
+      "project": "{{project2}}",
+      "resourceId": "https://dev.nexus.test.com/simplified-resource/2"
     },
     {
-      "@type" : "File",
-      "path" : "some/other/nexus-logo.png",
-      "project" : "{{project1}}",
-      "resourceId" : "https://dev.nexus.test.com/simplified-resource/logo"
+      "@type": "File",
+      "path": "/some/other/nexus-logo.png",
```

Comment:
the `project` field should always be present in the response, whether it is another project or the current project.